testui/support/testui_driver.py

```diff
@@ -18,7 +19,7 @@ class TestUIDriver:
     __test__ = False
 
 
-    def __init__(self, driver):
+    def __init__(self, driver: 'TestUIDriver'):
```

Comment:
Not sure what you want to do with this change

Reply:
Just a type annotation. If you don't quote it, Python says `"TestUIDriver" is not defined`